tests/e2e/test-build-js.R

```diff
@@ -1,2 +1,30 @@
 rhino::build_js()
-stopifnot(file.exists(fs::path("app", "static", "js", "app.min.js")))
+
+min_js_path <- fs::path("app", "static", "js", "app.min.js")
+index_js_path <- fs::path("app", "js", "index.js")
+
+testthat::expect_equal(
+  readLines(min_js_path, warn = FALSE),
+  "var App;App={};"
+)
+
+# Create minimal css and check app.min.css output
+cat(
+  "function sayHello() { console.log('Hello'); } export { sayHello };\n",
+  file = index_js_path,
+  append = TRUE

```

Comment:
Is `append` needed here?
```suggestion
  file = index_js_path
```